<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optiondlg</class>
 <widget class="QDialog" name="optiondlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>259</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_helperaddr">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>9</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoskipgc">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>95</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动跳过动画</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>人工地址:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autostartgame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>152</y>
     <width>119</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动开启游戏窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>38</y>
     <width>27</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>38</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>5797</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>67</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>鼠标移动间隔:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>96</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>第二次鼠标移动间隔:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>67</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>96</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>96</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autohide">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>125</y>
     <width>119</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>启动后直接最小化</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autorun">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>152</y>
     <width>95</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开机自动启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>67</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>optiondlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>optiondlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
